Core/Src/bms.c

```diff
@@ -41,23 +37,9 @@ void bms_init()
 	/*TO-DO: specify timer attributes*/
 	bms->bms_monitor_timer =
 		osTimerNew(&bms_fault_callback, osTimerOnce, NULL, NULL);
-
-	bms_monitor_queue =
-		osMessageQueueNew(CAN_QUEUE_SIZE, sizeof(can_msg_t), NULL);
-	assert(bms_monitor_queue);
 }
 
-void vBMSCANMonitor(void *pv_params)
+void handle_dcl_msg()
```

Comment:
can you confirm that if bms stops then the fault comes on, and most improtantly if the BMS returns to pperation the fault clears?  This can be simulated by re-flashing shep and seeing the light turn on then off.

Reply:
Confirmed